Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -295,6 +319,10 @@ Updates attributes of an existing Active Directory user.
 #### Base Command
 
 `ad-update-user`
+
+##### Required Permissions
+Requires `Create, delete, and manage user accounts` permissions.
```

Comment:
Is there an option to use Read/Write All user properties?

Reply:
I didnt find anything about this specific permission but `Create, delete, and manage user accounts` should be enough

Reply:
Same as above, only Read / Write all properties ( without delete) 
to `user objects`